### ISSUE TITLE
Error when stopping service

### ISSUE THREAD

**Issue by bmauger** [reykjavikingur/breakfast-butter#3]
When stopping Gulp (Ctrl-C), terminal does not return to a command prompt. Striking any key causes an error to be displayed. Ctrl-C must be used again to return to the command prompt.

```
^C
C02SH6Y0G8WP:responsive-scrambler bmauger$ npm testevents.js:163
      throw er; // Unhandled 'error' event
      ^
Error: read EIO
    at exports._errnoException (util.js:1050:11)
    at TTY.onread (net.js:582:26)
```

**Comment by rkiran16** [comment]
I am trying to install butter using command "butter install "  but i am seeing the following error. Any idea ?

installing dependencies, this may take awhile...events.js:160
      throw er; // Unhandled 'error' event
      ^

Error: spawn gulp ENOENT
    at exports._errnoException (util.js:1026:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:193:32)
    at onErrorNT (internal/child_process.js:359:16)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)

**Comment by bmauger** [comment]
Install Gulp globally. –g

**Comment by rkiran16** [comment]
worked . Thanks Mauger

**Comment by bmauger** [comment]
https://media.giphy.com/media/QN6NnhbgfOpoI/giphy.gif

**Comment by camdagr8** [comment]
Fixed in commit: 063da3af1e38a7360e371379f7fafa228cde3537
